src/components/CloudCard/index.js

```diff
@@ -15,7 +15,7 @@ function CloudCard({ title, subtitle, text, link }) {
           <h2>{subtitle}</h2>
         </div>
       </div>
-      <div className="cloud-card__inner-content">{text}</div>
+      <p className="cloud-card__inner-content">{text}</p>
```

Comment:
I agree this should be a <p>, it's a good change, but can it be inside the original div? So:
```
<div className="cloud-card__inner-context"><p>{text}</p></div>
```
Let me know what you think.

Reply:
Yeah that makes sense!